src/test/rmc-dependency-test/diamond-dependency/run-dependency-test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+# Test for platform
+platform=`uname -sp`
+if [[ $platform != "Linux x86_64" ]]; then
+  echo "Codegen script only works on Linux x86 platform"
```

Comment:
What happens on mac?

Reply:
With the given command, it fails to find `std`:

```
error[E0463]: can't find crate for `std`
  |
  = note: the `x86_64-unknown-linux-gnu` target may not be installed
  = help: consider downloading the target with `rustup target add x86_64-unknown-linux-gnu`
  = help: consider building the standard library from source with `cargo build -Zbuild-std`
```

If you try to build std (using nightly), you hit ICE's within building std. This block to skip is copied from the Firecrackers regression test.